### PR TITLE
WRITING-902 - fixes

### PR DESCRIPTION
**Pull request by mpaik**

### CODE REVIEW

File: source/reference/operator/update/addToSet.txt [hunk 1]
```diff
@@ -33,12 +33,36 @@ elements in the modified set.
...
+   {"_id": 1, "letters": ["a", "b"]}
+
+Calling
+
+.. code-block:: javascript
+
+ db.test.update(
+   {"_id": 1 },
+   {$addToSet: {"letters": ["c" , "d"]}}
+ )
```
Comment (tychoish) [inline source/reference/operator/update/addToSet.txt]: - not indented.
- quoting is inconsistent.

File: source/reference/operator/update/addToSet.txt [hunk 2]
```diff
@@ -33,12 +33,36 @@ elements in the modified set.
 If the field is absent in the document to update, :update:`$addToSet`
 creates the array field with the specified value as its element.
 
-If the field is **not** an array, the operation will fail.
+If the field is **not** an array, the operation will fail with an
+error message stating that :update:`$addToSet` cannot be applied to a
+non-array field.
```
Comment (tychoish) [inline source/reference/operator/update/addToSet.txt]: should either render the error message/response exactly here (which has pros and cons) or leave this more or less as it is.

File: source/reference/operator/update/addToSet.txt [hunk 3]
```diff
@@ -33,12 +33,36 @@ elements in the modified set.
...
+Calling
+
+.. code-block:: javascript
+
+ db.test.update(
+   {"_id": 1 },
+   {$addToSet: {"letters": ["c" , "d"]}}
+ )
+
+results in
```
Comment (tychoish) [inline source/reference/operator/update/addToSet.txt]: these fragment phrases without colons or capitalization are not consistent with our style, and we should tighten up these examples.

Comment (mpaik) [inline source/reference/operator/update/addToSet.txt]: By all means, knock yourself out.

**Comment by tychoish** [comment]
flatten and change commit message to not include references to WRITING tickets.

**Comment by schmalliso** [comment]
Merged!

**Pull request closed without merging**